docs/videos/running_all_devs.md

```diff
@@ -0,0 +1,41 @@
+# Running the All Core Devs call
+
+Create draft agenda file titled “alldevs[date].md” with the date of the next call in YYYYMMDD format.
+- Update the mkdocs.yml file of the wiki with a relative link to the file under the Videos > Meetings > All Core Devs > Agendas section.   
+- Add link to the agenda to the [Overview](dev.md) page in the wiki.
+
+Review proposed agenda items
+ - Check alldevs email address throughout the week for proposed agenda items. Review proposed items and respond with [accepted](all_devs_accepted.md) or [rejected](all_devs_rejected.md) templates.
+ - Add approved agenda items and the email addresses of agenda item owners to this meeting’s “Agenda + Guest Participants for [date]” email, replacing [date] with the date of the meeting.
+  - All approved agenda items to the next call’s agenda in the wiki.
+
+Ten mins before scheduled to begin. Create [Jitsi](https://meet.jit.si) channel with unique link.
+
+Five mins before scheduled to begin. Send email with all participants bcc’d.
+ - bcc “repeat participant” list.
+ - bcc guest participants from “Agenda + Guest Participants for [date]” email.
+ - Include in email:
+   - Unique link to Jitsi channel.
+   - Link to the agenda.
+
+Open Youtube and copy the Livestream key.
+ - Creator Studio -> Livestreaming -> Stream now -> Stream name/key.
+
+Start livestream in Jitsi when the meeting is scheduled to start.
+
+Welcome viewers and participants on the call. “Before we begin..” Designate a note taker if one has not volunteered already.
+ - Document meeting attendees, agenda items, notes from discussions.
+
+Do roll call and have everyone say a quick introduction of who they are.
+
+Run through each agenda item and pass to item owner to lead. Conclude each item discussion by repeating action items.
+
+Keep meeting running on time. The goal is to have all agenda items addressed by the scheduled end time.
+
+If all agenda items are covered before the scheduled end time, the meeting is done early.
+
+Thank participants and viewers for attending, then stop the livestream.
+
+Upload notes and video recording link to wiki.
+
+Create an [issue](https://github.com/aragon/aragon-wiki/issues) in the wiki titled "make a backup of video recording" and link to the call video in the description.
```

Comment:
Not sure what making the back up would entail... but do we need an new issue for each core devs call for this? If it's easy enough to do, could we do it immediately after the call?

Reply:
@sohkai it would entail going into youtube creator studio and clicking the "download" button then re-uploading wherever we end up archiving media (probably the Foundation's Nextcloud instance).